clr/commands.py

```diff
@@ -310,6 +313,10 @@ def arg_text(arg_name, default_value):
             for line in docstr.split('\n'):
                 print(text_wrapper.fill(line))
 
-    def cmd_argtest(self, a, b, c=4, d=None, e=False):
+    def cmd_argtest(self, a, b, c=4, d=None, e=False, g=True):
+        """For testing arg parsing."""
+        print(f'a={a} b={b} c={c} d={d} e={e} g={g}')
```

Comment:
Can't wait for Python 3.8 :) 

```python
print(f'{a=} {b=} {c=} {d=} {e=} {g=}')
```

https://docs.python.org/3/whatsnew/3.8.html#f-strings-support-for-self-documenting-expressions-and-debugging